src/url.js

```diff
@@ -179,8 +180,15 @@ export function serializeQueryString(params) {
     const v = params[k];
     if (v == null) {
       continue;
+    } else if (isArray(v)) {
+      for (let i = 0; i < v.length; i++) {
+        const sv = /** @type {string} */ (v[i]);
+        s.push(`${encodeURIComponent(k)}=${encodeURIComponent(sv)}`);
```

Comment:
I believe the `[]` in your example is just a way of naming the parameters. But no tested this out and the default browser behavior are turned into `k=v1&k=v2`. If you name your inputs `k[]` then yours would be the case.